### DISCUSSION TITLE
Mouse with absolute coordinates

### DISCUSSION THREAD

**Discussion by krucho** [adafruit/Adafruit_TinyUSB_Arduino#211] [Q&A]
Hi everyone!
I've been playing with a RP2040 as HID Mouse with Arduino IDE, but my application needs absolute coordinates. Based on this [thread](https://github.com/hathach/tinyusb/pull/1363/files/8c79b8924eabf441f40a8ce3cab8828688575b6d) I modified some files and make a basic sketch to work, the problem is, it only works with int8_t for X and Y (-128 to 127), actually from 0 to 127 on the screen.
Is there a way to "upgrade" the X and Y to int16_t or uint16_t without breaking everything?

**Comment by hathach** [comment]
you can create your own 16-bit report descriptor
